benefits/eligibility/templates/eligibility/start.html

```diff
@@ -21,8 +21,43 @@ <h1>{{ headline_text }}</h1>
 {% block inner-content %}
   <p class="py-4">{% translate "You will need a few items to continue:" %}</p>
   <ul class="d-flex flex-column gap-4 list-unstyled ps-0 mb-0">
-    {% include eligibility_item_template %}
-    {% include "eligibility/includes/eligibility-item--contactless-card--start.html" %}
+    {% if eligibility_item_template %}
+      {% include eligibility_item_template %}
+    {% endif %}
```

Comment:
This can be removed now right? Because of the change in https://github.com/cal-itp/benefits/pull/2774/commits/cfb6a3313edec50bf23caa21a451d15f8b3a5988 to remove `eligibility_item_template` from `EligibilityStart`

Reply:
Oops, yes you are right -- thanks for the catch.